benchmarks/stability/aggregate.py

```
@@ -0,0 +1,100 @@
+
+    # Defer heavy imports until we are sure we have files to process
+    from mlip_arena.tasks.stability.analysis import gather_results
+    from mlip_arena.tasks.stability.data import get_atoms_from_db
+
+    compositions = []
+    try:
+        for atoms in get_atoms_from_db("random-mixture.db"):
+            if len(atoms) == 0:
+                continue
+            compositions.append(atoms.get_chemical_formula())
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)